pkg/apisix/resource.go

```diff
@@ -72,6 +72,27 @@ type item struct {
 	Value json.RawMessage `json:"value"`
 }
 
+type routeWrap v1.Route
```

Comment:
You can just create a type `Vars` in `pkg/types/apisix/v1`, and Implement the `json.Unmarshaler` interface for it.

Reply:
Yes, thanks for your suggestion! I have rewrite.